app/frontend/src/common/store/auth.js

```diff
@@ -29,7 +29,7 @@ const auth = {
               state.keycloak.hasRealmRole('registries_statutory_authority'))
           },
           wells: {
-            view: (!!state.keycloak.hasRealmRole('gwells_admin') || !!state.keycloak.hasRealmRole('wells_viewer')),
+            view: !!state.keycloak.hasRealmRole('wells_viewer'),
```

Comment:
No argument from me about going in this direction but think it would be a good idea to review/update the other permissions checks at the same time.

In the meantime, I have updated the role mapping for the `gwells_admin` keycloak groups in dev/test/prod.